web/api/src/lib/starknet.ts

```diff
@@ -87,7 +87,7 @@ export async function notarySubmitProfile(
   await erc20Approve(notary, ZorroAddress, depositSize)
   console.log('erc20 approved')
 
-  console.log('attempting zorro submit')
+  console.log('attempting zorro submit', { calldata: [cid, ethereumAddress] })
```

Comment:
(tiny nit) could defining this calldata array separately result in what is logged getting out of sync with what happens in the actual `addTransaction` call?

Reply:
Yeah just needed this for some immediate debugging anyway, removed.